### PR TITLE
DM-11664: Flake8 fixes

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: python/lsst/daf/persistence/butlerLocation.py [hunk 1]
```diff
@@ -89,24 +87,23 @@ def __init__(self, datasetType, obj, setter, getter, subset, inputOnly):
...
+             self.disassembler,
+             self.python,
+             self.dataId,
+             self.mapper,
+             self.componentInfo,
+             self.repository)
 
         def __repr__(self):
             return "ComponentInfo(datasetType=%s, obj=%s, setter=%s, getter=%s)" % (
                 self.datasetType, self.obj, self.setter, self.getter)
```
Comment (n8pease) [inline python/lsst/daf/persistence/butlerLocation.py]: this is the definition to remove.

Comment (timj) [reply python/lsst/daf/persistence/butlerLocation.py]: I think there's another one at line 146...

File: python/lsst/daf/persistence/butlerLocation.py [hunk 2]
```diff
@@ -89,23 +87,18 @@ def __init__(self, datasetType, obj, setter, getter, subset, inputOnly):
...
-                self.dataId,
-                self.mapper,
-                self.componentInfo,
-                self.repository)
-
-        def __repr__(self):
-            return "ComponentInfo(datasetType=%s, obj=%s, setter=%s, getter=%s)" % (
-                self.datasetType, self.obj, self.setter, self.getter)
-
+        return 'ButlerComposite(assembler:%s, disassembler:%s, python:%s, dataId:%s, mapper:%s, ' \
```
Comment (timj) [inline python/lsst/daf/persistence/butlerLocation.py]: Of the three, I kept this one as it looked the most detailed.

File: tests/test_repositoryArgs.py [hunk 3]
```diff
@@ -122,12 +120,12 @@ def testAbsoluteFilePathWithSchema(self):
         uri = urllib.parse.urljoin('file://', self.testDir)
         args = dp.RepositoryArgs(root=uri, mapper='lsst.daf.persistence.Mapper')
         butler = dp.Butler(outputs=args)
+        self.assertIsInstance(butler, dp.Butler)
         self.assertTrue(os.path.exists(os.path.join(self.testDir, 'repositoryCfg.yaml')))
 
-        butler2 = dp.Butler(inputs=uri)
```
Comment (timj) [inline tests/test_repositoryArgs.py]: I removed this since it seemed to be the same call as the next one but not protected by a try block.

**Pull request merged**